doc/source/limitation.rst

```diff
@@ -33,44 +24,40 @@ Query log cannot be directed to table.
-  ``LOAD DATA INFILE`` processing will commit every 10 000 rows.
-  So large transactions due to ``LOAD DATA``
-  will be split to series of small transactions.
+Maximum allowed transaction size is defined by the |max_ws_rows| and |max_ws_size| variables.
+   ``LOAD DATA INFILE`` processing will commit every 10 000 rows.  So large
+   transactions due to ``LOAD DATA`` will be split to series of small
+   transactions.
 
-* Due to cluster-level optimistic concurrency control,
-  transaction issuing ``COMMIT`` may still be aborted at that stage.
-  There can be two transactions writing to the same rows
-  and committing in separate |PXC| nodes,
-  and only one of the them can successfully commit.
-  The failing one will be aborted.
-  For cluster-level aborts, |PXC| gives back deadlock error code: ::
+Transaction issuing ``COMMIT`` may still be aborted at that stage.
+   Due to cluster-level optimistic concurrency control. There can be two
+   transactions writing to the same rows and committing in separate |PXC| nodes,
+   and only one of the them can successfully commit. The failing one will be
+   aborted. For cluster-level aborts, |PXC| gives back deadlock error code: ::
 
    (Error: 1213 SQLSTATE: 40001  (ER_LOCK_DEADLOCK)).
 
-* XA transactions are not supported due to possible rollback on commit.
-
-* The write throughput of the whole cluster is limited by the weakest node.  If
-  one node becomes slow, the whole cluster slows down.  If you have requirements
-  for stable high performance, then it should be supported by corresponding
-  hardware.
+XA transactions are not supported
+   Due to possible rollback on commit.
 
-* The minimal recommended size of cluster is 3 nodes.  The 3rd node can be an
-  arbitrator.
+Write throughput of the whole cluster is limited by the weakest node.
+   If one node becomes slow, the whole cluster slows down.  If you have
+   requirements for stable high performance, then it should be supported by
+   corresponding hardware.
 
-* InnoDB fake changes feature is not supported.
+Minimal recommended size of cluster is 3 nodes.
+   The 3rd node can be an arbitrator.
 
-* ``enforce_storage_engine=InnoDB`` is not compatible with
-  ``wsrep_replicate_myisam=OFF`` (default).
+``enforce_storage_engine=InnoDB`` is not compatible with ``wsrep_replicate_myisam=OFF``
+   :variable:`wsrep_replicate_myisam` is set to ``OFF`` by default.
 
-* When running |PXC| in cluster mode,
-  avoid ``ALTER TABLE ... IMPORT/EXPORT`` workloads.
-  It can lead to node inconsistency if not executed in sync on all nodes.
+Avoid ``ALTER TABLE ... IMPORT/EXPORT`` workloads when running |PXC| in cluster mode.
+   It can lead to node inconsistency if not executed in sync on all nodes.
 
-* All tables must have the primary key. This ensures that the same rows appear
-  in the same order on different nodes. The ``DELETE`` statement is not supported on
-  tables without a primary key.
+All tables must have the primary key.
```

Comment:
"must have the primary key" -> "must have a primary key"

Reply:
there can be only one primary key (it is still one even if it is made of values of multiple fields). For this reason i would still suggest to keep 'the'

Reply:
using 'the' sounds like there is a single primary key somewhere that all tables are sharing. it sounds awkward.